soroban-env-host/src/e2e_invoke.rs

```diff
@@ -63,6 +65,7 @@ pub struct InvokeHostFunctionRecordingModeResult {
     pub invoke_result: Result<ScVal, HostError>,
     /// Resources recorded during the invocation, including the footprint.
     pub resources: SorobanResources,
+    pub restored_rw_entry_ids: Vec<u32>,
```

Comment:
Do you mind adding a comment for this? It's not immediately clear that it's storing indices of entries in the readWrite set. Maybe it should be called `restored_rw_entry_indices` instead?